lib/openstax/aws/msk_cluster.rb

```diff
@@ -0,0 +1,19 @@
+module OpenStax::Aws
+  class MskCluster
+
+    attr_reader :client, :msk_instance_identifier
+
+    delegate_missing_to :@client
+
+    def initialize(msk_instance_identifier:, region:)
+      @msk_instance_identifier = msk_instance_identifier
+      @client = ::Aws::Kafka::Client.new(region: region)
+    end
+
+    def bootstrap_hosts()
```

Comment:
you dont need () here.  

https://github.com/airbnb/ruby#no-args-parens

Reply:
python bleed through, probably. :)